utils/geocode_geoareas.py

```
@@ -0,0 +1,1105 @@
+            # Create lookup from join results
+            join_lookup = {}
+            try:
+                with arcpy.da.SearchCursor(temp_places_join, join_fields) as join_cursor:
+                    for row in join_cursor:
+                        oid = row[0]
+                        place_name = row[1] if len(row) > 1 and row[1] else None
+                        place_fips = row[2] if len(row) > 2 and row[2] else None
+                        
+                        if place_name:  # Only store if we got a place
+                            join_lookup[oid] = (place_name, place_fips)
+            except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)